astropy/coordinates/angle_utilities.py

```diff
@@ -520,14 +520,15 @@ def sexagesimal_to_string(values, precision=None, pad=False, sep=(':',),
     interface to this functionality.
     """
 
-    # If the coordinates are negative, we need to take the absolute value of
-    # the (arc)minutes and (arc)seconds. We need to use np.abs because abs(-0)
-    # is -0.
-    values = (values[0], np.abs(values[1]), np.abs(values[2]))
+    # Check to see if values[0] is negative, using np.copysign to handle -0
+    sign = np.copysign(1.0, values[0])
+    # If the coordinates are negative, we need to take the absolute values.
+    # We use np.abs because abs(-0) is -0
+    # TODO: Is this true? (MHvK, 2018-02-01: not on my system)
```

Comment:
Does this "TODO" means this is still a work in progress?

Reply:
No, just that at some point someone should figure out if this really is a problem. But I didn't want to hold up the fix just for that...